search-api.md

```diff
@@ -23,12 +23,6 @@ After receiving a request, the remote server can respond in one of two ways:
   * If a compatible version (`vX.Z` where `Z>=Y`) is supported by the remote server, it should provide a response using this version.
   * If no appropriate version is supported by the remote server, it should respond with `Not Acceptable (406)`, containing a JSON body with a description of the error. All responses, including this one, should contain a `Content-Type` header with the latest API version supported by the server. This will enable the user to re-submit the request using this version of the API.
```

Comment:
I see now what you're complaining about @allasm. I'll try to clarify that now. 

@sdumitriu, I defined compatible in that line as same major version and at least that minor version (`vX.Z` is a compatible response to `vX.Y` where `Z>=Y`). Then, any minor releases should only include additional fields (or remove optional fields), which can then be ignored in a backwards-compatible way.